soundata/datasets/tau_nigens_sse_2021.py

```diff
@@ -0,0 +1,652 @@
+    :class: dropdown
+
+    *TAU NIGENS Spatial Sound Events: scene recordings with (moving) sound events of distinct categories*
+
+    *Created By:*
+        Archontis Politis, Sharath Adavanne, Tuomas Virtanen.
+        Audio Research Group, Tampere University (Finland). Version 1.2.0
+
+    *Description:*
+        DESCRIPTION:
+
+        Spatial sound-scene recordings, consisting of sound events of distinct categories 
+        in a variety of acoustical spaces, and from multiple source directions and distances.
+        The spatialization of all sound events is based on filtering through real spatial 
+        room impulse responses (RIRs) of diverse acoustic environments. The sound events are 
+        spatialized as either stationary sound sources, or moving sound sources, in which case 
+        time-variant RIRs are used. 
+
+        Each scene recording is delivered in microphone array (MIC) and first-order Ambisonics (FOA) 
+        format. 
+
+    *Audio Files Included:*
+        * 600 one-minute-long sound scene recordings with annotations (development dataset).
+        * 200 one-minute-long sound scene recordings without annotations (evaluation dataset).
+        * Sampling rate is 24 kHz (16-bit signed integer PCM).
+        * About 500 sound event samples distirbuted over 12 target classes.
+        * About 400 sound event samples used as interference events.
+        * 1st order HOA or tetrahedral microphone array formats.
+        * Realistic spatialization and reverberation through multichannel RIRs collected in 13 different enclosures.
+        * From 1184 to 6480 possible RIR positions across the different rooms.
+        * Both static reverberant and moving reverberant sound events.
+        * Three possible angular speeds for moving sources of approximately 10, 20, or 40deg/sec.
+        * Up to three overlapping sound events possible, temporally and spatially.
+        * Simultaneous directional interfering sound events with their own temporal activities, static or moving.
+        * Realistic spatial ambient noise collected from each room is added to the spatialized sound events, at varying signal-to-noise ratios (SNR) ranging from noiseless (30dB) to noisy (6dB) conditions.
+
+    *Annotations Included:*
+        * Each recording in the development set has labels of events and DoAs in a plain csv file with the same filename.
+        * Each row in the csv file has a frame number, active class index, event number index, azimuth, and elevation.
+        * Frame, class, and track enumeration begins at 0. 
+        * Frames correspond to a temporal resolution of 100msec. 
+        * Azimuth and elevation angles are given in degrees, rounded to the closest integer value, with azimuth and elevation being zero at the front, azimuth $\phi \in [-180^{\circ}, 180^{\circ}]$, and elevation $\theta \in [-90^{\circ}, 90^{\circ}]$. Note that the azimuth angle is increasing counter-clockwise ($\phi = 90^{\circ}$ at the left).
```

Comment:
Same comment as the other loader, the equations don't render properly in the docs ([see here](https://soundata--77.org.readthedocs.build/en/77/source/soundata.html#tau-nigens-sse-2021))

Reply:
Do we know if we support LaTex at all?